TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/BlockingTrafficSourceTest.java

```diff
@@ -69,7 +69,7 @@ void readNextChunkTest() throws Exception {
         blockingSource.stopReadsPast(sourceStartTime.plus(Duration.ofMillis(nStreamsToCreate)));
         var exception = Assertions.assertThrows(ExecutionException.class,
                 ()->blockingSource.readNextTrafficStreamChunk(rootContext::createReadChunkContext)
-                        .get(10, TimeUnit.MILLISECONDS));
+                        .get(10000, TimeUnit.MILLISECONDS));
```

Comment:
Ran into some timeouts with concurrent runs when this was set to 10

Reply:
This is a big difference.  Timing is a painful thing to test, but this was making an attempt at making sure that the next chunk would stop waiting relatively quickly due to the signal from stopReadsPast() rather than just simply timing out.  If you make the timeout value something bigger than 10s, this might be fair (maybe it even is already).

Reply:
Do you have a suggestion here? Just trying to prevent a flaky test but also don't want to try and repro the tons of test runs I did previously that stumbled on an occasional failure here. I'm happy moving it to like 1 second or half a second.

Reply:
I just looked at the next line.  Since it's EOF, any timeout induced return wouldn't be signaling EOF, so this seems fine with any value.  If it takes us 10ms or 1s to get the EOF, that's fine, especially from a functional perspective.